internal/brim/filter/filter_test.go

```diff
@@ -356,37 +427,44 @@ func prepareVersionMocks(bucket, key, access, secret string, fetcherMock *versio
 	}
 }
 
-func generateAkubraConfig(numberOfStorages int) *config.Config {
+func generateAkubraConfig(numberOfShards int, numberOfStoragesPerShard int) *config.Config {
 	akubraConfig := &config.Config{}
 	akubraConfig.Storages = storagesConfig.StoragesMap{}
 	akubraConfig.Shards = storagesConfig.ShardsMap{}
 
-	akubraConfig.ShardingPolicies = regionsConfig.ShardingPolicies{}
-	akubraConfig.ShardingPolicies["test"] = regionsConfig.Policies{
-		Domains: []string{"localhost"},
-		Shards:  []regionsConfig.Policy{{Weight: 1, ShardName: "test"}},
-	}
+	weightPerShard := float64(1) / float64(numberOfStoragesPerShard)
```

Comment:
When value of `numberOfStoragesPerShard` will be zero result of below calculation is `+Inf`.
Maybe we need special IF condition for this case ?

Reply:
There's no test that defines it as zero.